lib/index.js

```diff
@@ -185,6 +185,11 @@ const pickManifest = (packument, wanted, opts) => {
 
 module.exports = (packument, wanted, opts = {}) => {
   const mani = pickManifest(packument, wanted, opts)
+  // Decorate with publish time if available
+  const _time = mani && packument.time && packument.time[mani.version] || null
```

Comment:
```suggestion
  const _time = packument.time.?[mani?.version]]
```

See line 194 where we already assume `packument` is defined.

`|| null` doesn't do much, as it sets to null only if it's falsey but then we turn around and check it again for truthiness in `if (_time)`